src/navigation/TabNavigation.tsx

```diff
@@ -0,0 +1,39 @@
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {HomeScreen} from 'screens';
+import SettingsNavigation from './SettingsNavigation';
+import { blueColors, grayColors } from 'styles/themeColors';
+
+const TabNavigation = () => {
+  const Tab = createBottomTabNavigator();
+
+  return (
+    <Tab.Navigator
+      initialRouteName="Weather"
+      screenOptions={({route}) => ({
+        tabBarIcon: () => {
+          let iconName = 'ios-information-circle';
+          if (route.name === 'Weather') {
```

Comment:
let's add a global navigation enum in which we will place all screen namings like:
export enum PageNames {
Weather = Weather
}
and reuse PageNames everywhere in your app